Makefile

```diff
@@ -1,13 +1,15 @@
-build:
```

Comment:
You can add a self descriptive help as on all our projects at marmelab too.

help:
	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'